bubblesort.rb

```diff
@@ -0,0 +1,18 @@
+def bubble_sort(array)
+  return array if array.length <= 1
+
+  sum1 = array.length - 1
+  loop do
+    break if sum1 <= 0
+
+    sum1 -= 1
+    array.each do |n|
+      x = array.index(n)
+      next unless array[x + 1]
+
+      array[x], array[x + 1] = array[x + 1], array[x] if n > array[x + 1]
+    end
+  end
+  array
+end
+p bubble_sort([4, 3, 78, 2, 0, 2])
```

Comment:
In line 18 you have this code `p bubble_sort([4, 3, 78, 2, 0, 2])` this code is debugging code. You use it to test if the code works but that's debugging code. 
Please delete it or if you want, keep it in another branch.